elastic/security/templates/composable/logs-endpoint.events.file.json

```diff
@@ -38,6 +36,9 @@
     "data_stream": {
       "hidden": false,
       "allow_custom_routing": false
+      {% if build_flavor != "serverless" %},
+      "failure_store": false
+      {% endif %}
```

Comment:
That's a snapshot build feature, so please remove.

Reply:
Done